webapp/components/PostCard/PostCard.vue

```diff
@@ -132,7 +132,7 @@ export default {
       )
     },
     isPinned() {
-      return this.post && this.post.pinnedBy
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-21T22:12:40Z" title="Tuesday, January 21st 2020, 11:12:40 pm +01:00">Jan 21, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -132,7 +132,7 @@ export default {
       )
     },
     isPinned() {
-      return this.post && this.post.pinnedBy
```


I added missing setup to `posts` factory. @mattwr18 @aonomike do we want to change implementation here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-22T09:16:44Z" title="Wednesday, January 22nd 2020, 10:16:44 am +01:00">Jan 22, 2020</time>_

---


I thought the idea was to use `this.post.pinned` instead of pinnedBy... see line `135` 
The reason is that `pinnedBy` is an author with nested attributes, ie more expensive request.. whereas, pinned is an attribute.
The reason it was `pinnedBy` in the first place is that I hadn't full taken into account the performance on production data, then you jumped in and fixed it by adding the `pinned` attribute, then he forgot to refactor this function.